src/pybind/mgr/dashboard/tox.ini

```diff
@@ -11,10 +11,10 @@ setenv=
     WEBTEST_INTERACTIVE = false
     LD_LIBRARY_PATH = {toxinidir}/../../../../build/lib
     PATH = {toxinidir}/../../../../build/bin:$PATH
+    COVERAGE_FILE = .coverage.{envname}
+    UNITTEST = true
```

Comment:
why do you think we need to have `UNITTEST=true` for lint?

Reply:
I was just being conservative. The lint env does not need the UNITTEST var.

If make check passes, I'll revert the UNITTEST change, and run make check again.

Reply:
the UNITTEST env var is used by the test itself, and i think pylint just tries to analyze the python code instead of evaluating it. that's why i don't think it's necessary to `export UNITTEST=true` here.

Reply:
```
Traceback (most recent call last):
  File "/usr/bin/tox", line 9, in <module>
    load_entry_point('tox==1.4.2', 'console_scripts', 'tox')()
  File "/usr/lib/python2.7/site-packages/tox/_cmdline.py", line 24, in main
    config = parseconfig(args, 'tox')
  File "/usr/lib/python2.7/site-packages/tox/_config.py", line 23, in parseconfig
    parseini(config)
  File "/usr/lib/python2.7/site-packages/tox/_config.py", line 174, in __init__
    self._makeenvconfig("python", "_xz_9", reader._subs, config)
  File "/usr/lib/python2.7/site-packages/tox/_config.py", line 221, in _makeenvconfig
    vc.commands = reader.getargvlist(section, "commands")
  File "/usr/lib/python2.7/site-packages/tox/_config.py", line 331, in getargvlist
    commandlist.append(self._processcommand(current_command))
  File "/usr/lib/python2.7/site-packages/tox/_config.py", line 351, in _processcommand
    new_word = re.sub(pat, self._replace_match, word)
  File "/usr/lib64/python2.7/re.py", line 151, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "/usr/lib/python2.7/site-packages/tox/_config.py", line 485, in _replace_match
    return handler(match)
  File "/usr/lib/python2.7/site-packages/tox/_config.py", line 447, in _replace_env
    (envkey, envkey))
tox.ConfigError: ConfigError: substitution env:'COVERAGE_FILE': unkown environment variable 'COVERAGE_FILE'
```
seems teh env var is not found even we set it unconditionally..